lib/cookieman.js

```diff
@@ -114,10 +114,27 @@ function clearAll (name) {
   return cleared
 }
 
+/**
+ * gets the value of a cookie, or returns null
+ *
+ * @public
+ * @param {String} name The cookie name
+ * @returns {Null} or {Boolean} Returns null if the cookie is not found or the cookie's value
+ */
+function cookieValue (name) {
+  var val = null
+  var cookies = getCookie(name)
+  if (cookies && cookies[0] && cookies[0].value) {
```

Comment:
If the cookie is set to `''` then this function will return `null`. This means you cannot tell if a cookie is empty or not set at all but I don't really know if that matters...